dev/com.ibm.ws.injection.core/src/com/ibm/ws/injectionengine/processor/ResourceProcessor.java

```diff
@@ -948,7 +959,7 @@ private boolean isManagedBeanRef(Class<?> injectType, String injectTypeName) {
         boolean result;
 
         if (injectType != null && injectType != Object.class) {
-            result = injectType.isAnnotationPresent(ManagedBean.class);
+            result = (MANAGED_BEAN_CLASS != null) ? injectType.isAnnotationPresent(MANAGED_BEAN_CLASS) : false;
```

Comment:
Can this be changed to:

```
if (MANAGED_BEAN_CLASS == null) {
    result = false;
} else if (injectType != null && injectType != Object.class) {
    result = injectType.isAnnotationPresent(MANAGED_BEAN_CLASS);
} else {
...
```

Or do we have to do the else even if ManagedBean class isn't found?

Reply:
No; the else would always return false, so it is clearer to just return false if the class is null.
Changed as requested.